src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsInterstitialObject.java

```diff
@@ -80,23 +83,21 @@ public void closeInterstitial() throws InterruptedException {
 
     boolean isMercury = false;
     try {
-      isMercury = interstitialCloseButtonMercury.isDisplayed();
+      isMercury = interstitialCloseButtonWrapperMercury.isDisplayed();
       Log.log("Interstitial Mercury", "Yes", true);
     } catch (NoSuchElementException e) {
       Log.log("Interstitial Mercury", "No", true);
     }
 
     if (isMercury) {
-      String closeButtonText = interstitialCloseButtonMercury.getText();
-      if (closeButtonText.length() > 0) {
-        Integer waitTillCloseButtonAppears = Integer.parseInt(closeButtonText);
-        Log.log("Wait time for close button", String.valueOf(waitTillCloseButtonAppears), true);
-        Thread.sleep((waitTillCloseButtonAppears + WAIT_BUTTON_DELAY_TOLERANCE) * 1000);
-      }
+      String closeButtonText = interstitialCloseButtonWrapperMercury.getText();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "closeButtonText" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)